src/components/multi-action-button/multi-action-button.component.tsx

```diff
@@ -15,12 +21,16 @@ import {
 import { defaultFocusableSelectors } from "../../__internal__/focus-trap/focus-trap-utils";
 import Logger from "../../__internal__/utils/logger";
 
-const marginPropTypes = filterStyledSystemMarginProps(
-  styledSystemPropTypes.space
-);
-
 let deprecatedWarnTriggered = false;
 
+export interface MultiActionButtonProps
+  extends Omit<SplitButtonProps, "buttonType"> {
+  /** Button type: "primary" | "secondary" | "tertiary" */
+  buttonType?: "primary" | "secondary" | "tertiary";
+  /** Second text child, renders under main text, only when size is "large" */
+  subtext?: string;
+}
+
 const MultiActionButton = ({
```

Comment:
```suggestion
export const MultiActionButton = ({
```

I don't think it picks up the props in storybook without it